src/resolvers/collection.js

```diff
@@ -86,22 +86,28 @@ export default {
     },
     services: async (source, args, context, info) => {
       const {
-        associations = {}
+        association_details: associationDetails = {}
       } = source
 
       const { dataSources } = context
 
-      const { services = [] } = associations
+      const { services = [] } = associationDetails
+
+      const serviceConceptIds = []
+
+      services.forEach((service) => {
+        const { concept_id: serviceConceptId } = service
+        serviceConceptIds.push(serviceConceptId)
+      })
```

Comment:
This can be  simplified
```suggestion
      const serviceConceptIds = services.map((service) => service["concept_id"])
```